src/main/java/org/gitlab4j/api/ProtectedBranchesApi.java

```diff
@@ -150,23 +151,43 @@ public ProtectedBranch protectBranch(Object projectIdOrPath, String branchName,
      * @param mergeAccessLevel access levels allowed to merge (defaults: 40, maintainer access level)
      * @param unprotectAccessLevel access levels allowed to unprotect (defaults: 40, maintainer access level)
      * @param codeOwnerApprovalRequired prevent pushes to this branch if it matches an item in the CODEOWNERS file. (defaults: false)
+     * @param allowForcedPush by default a forced push is only prohibited on main/master, so a good default is to set this to false to disallow a forced push (and loosing all prior history of the former pushes)
      * @return the branch info for the protected branch
      * @throws GitLabApiException if any exception occurs
      */
     public ProtectedBranch protectBranch(Object projectIdOrPath, String branchName,
             AccessLevel pushAccessLevel, AccessLevel mergeAccessLevel, AccessLevel unprotectAccessLevel,
-            Boolean codeOwnerApprovalRequired) throws GitLabApiException {
-        Form formData = new GitLabApiForm()
+            Boolean codeOwnerApprovalRequired, Boolean allowForcedPush) throws GitLabApiException {
+        GitLabApiForm formData = new GitLabApiForm()
                 .withParam("name", branchName, true)
                 .withParam("push_access_level", pushAccessLevel)
                 .withParam("merge_access_level", mergeAccessLevel)
                 .withParam("unprotect_access_level", unprotectAccessLevel)
                 .withParam("code_owner_approval_required", codeOwnerApprovalRequired);
+        if (Objects.nonNull(allowForcedPush)) {
```

Comment:
The `if` here is not necessary. The method `.withParam(..)` is already taking care of that.

Reply:
Fixed with 7ad47ca0769bc88867208a1e2756d5e8d266e5fa